src/core/init/load_on_media_source.ts

```diff
@@ -176,7 +176,7 @@ export default function createMediaSourceLoader({
             return EMPTY;
           case "needs-source-buffer-flush":
             if (mediaElement.currentTime + 0.001 < mediaElement.duration) {
-                log.info("Init: Flushing mediaElement by performing a small seek.");
+                log.info("Init: Flushing audio buffer by performing a small seek.");
```

Comment:
We may adapt that event for something else than audio in the future.
As nothing in that event and handling says "audio" we might want to stay generic (e.g. `Flusing buffers`)